### ISSUE TITLE
Add `aria-label` to inputs

### ISSUE THREAD

**Issue by wojtekmaj** [wojtekmaj/react-date-picker#158]
> For accessibility reasons, each of the inputs generated should have a label.
> 
> Adding `aria-label='hour'`to the hour input, and one for minute, and PM/AM should be on by default, or give us the option to enable them.

Originally posted by @nickbristow in https://github.com/wojtekmaj/react-time-picker/issues/21#issue-445069045

**Comment by wojtekmaj** [comment]
Will introduce:

* `yearAriaLabel`
* `monthAriaLabel`
* `dayAriaLabel`

**Comment by wojtekmaj** [comment]
Closed via 919a9a1ec957bf70f7dfe21b6e88831e732969a5
